.github/workflows/push-trigger.yml

```diff
@@ -0,0 +1,61 @@
+name: admin-ui build upon a push
+
+on:
+  release:
+    types: [published]
+  pull_request:
+    types: [opened]
+  push:
+    branches:
+      - '!release-branch'
+      - release-1*
+      - master
+      - 1.0.*
+      - develop
+      - 1.1.*
+      - MOSIP-11719
+      - 1.2.*
+      - MOSIP*
```

Comment:
Please make only a single entry i.e. `- *`

Reply:
Done with the Change.